bin/zhtautau/runZHTauTauAnalysis.cc

```diff
@@ -1058,7 +1058,7 @@ int main(int argc, char* argv[])
     LorentzVector elDiff(0,0,0,0);
     for(size_t ilep=0; ilep<leptons.size(); ilep++){
       bool passKin(true),passId(true),passIso(true);
-      bool passLooseLepton(true), passSoftMuon(true), passSoftElectron(true), passVetoElectron(true);
+      bool passVeryLooseLepton(true), passLooseLepton(true), passSoftMuon(true), passSoftElectron(true), passVetoElectron(true);
```

Comment:
@clacaputo can you cross-check what I have done here correponds to what we discussed this morning? Thanks